pepper-apis/src/main/java/org/broadinstitute/ddp/log/SlackAppender.java

```diff
@@ -67,26 +67,6 @@ private void init(String slackHookUrl,
                       String slackChannel,
                       Integer queueSize,
                       Integer intervalInMillis) {
-        httpClient = HttpClient.newHttpClient();
```

Comment:
We might have a config object, but that doesn't mean it'll have slack configuration properties. So we're moving these inside the `if` block below.